manifests/config.pp

```diff
@@ -35,6 +35,8 @@
   $nova_password = undef,
   $neutron_password = undef,
   $neutron_shared_secret = undef,
+  $neutron_core_plugin = undef,
```

Comment:
Could this default to 'neutron.plugins.ml2.plugin.Ml2Plugin' to maintain compatibility with the old version?